pallet-logion-loc/src/lib.rs

```diff
@@ -86,14 +86,28 @@ impl<AccountId, LocId, EthereumAddress> Default for Requester<AccountId, LocId,
     }
 }
 
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, TypeInfo, Copy)]
+pub enum SupportedAccountId<AccountId, EthereumAddress> {
+    None, // Enables "null" account ID
+    Polkadot(AccountId),
+    Ethereum(EthereumAddress)
+}
+
+impl<AccountId, EthereumAddress> Default for SupportedAccountId<AccountId, EthereumAddress> {
+
+    fn default() -> SupportedAccountId<AccountId, EthereumAddress> {
+        SupportedAccountId::None
```

Comment:
Not sure to see a case for the value `None` and trait `Default`.

Reply:
This is required to make `SupportedAccountId` storable. Neither `AccountId` nor `EthereumAddress` have default values (for security reasons BTW) so I had to introduce `None` as a kind of "null object".